kinder/ci/kubeadm-periodic.tests.md

```diff
@@ -101,3 +101,11 @@ Kubeadm join and upgrade tests that ensure that kubeadm tolerates missing addon
 | from                     | e.g.              | to                       | e.g.             |
 | -------------------------| ----------------- | -------------------------| ---------------- |
 | master<br />(ci/latest)  | v1.19.0-alpha..   | master<br />(ci/latest)  | v1.19.0-alpha... |
+
+### Tests for external CA
+
+Kubeadm tests that ensure kubeadm support "init" and "join" in external CA mode (missing ca.key).
+
+| from                     | e.g.              | to                       | e.g.             |
```

Comment:
Should this be:
```
| Version                                | e.g.              |
| -------------------------------------- | ------            |
| master<br />(ci/latest)                | v1.20.0-alpha...  |
| current<br />(ci/latest-1.19)          | v1.19.2-alpha...  |
| current-1<br />(ci/latest-1.18)          | v1.18.x-alpha...  |
| current-2<br />(ci/latest-1.17)          | v1.17.x-alpha...  |
```
NB1. just noticing that discovering and kustomize test list should be amended as well to use current, current -1 instead of repeating master on many lines.
NB2. should we replace master with main?

Reply:
I think we should remove the tables under each test type. I do not think anyone else cares about this detail and we already know what we have looking at the yaml files.

Reply:
kk